### PR TITLE
Revert "Revert "Add new Synapse teams hierarchy""

### PR DESCRIPTION
**Pull request by ahayden**
Reverts Sage-Bionetworks/synapse-login-scipool#10

Let's put the Synapse teams maps back in.

### CODE REVIEW

**Review by zaro0508 — COMMENTED** [review]
i don't undestand.  this looks like https://github.com/Sage-Bionetworks/synapse-login-scipool/pull/8
however you said that broke the login app.  So are you going to break the login app again?

**Comment by ahayden** [comment]
I reverted because the app was broken, but I don't think this change did the breaking.

File: src/main/resources/scipooldev.properties [hunk 1]
```diff
@@ -1,5 +1,7 @@
 SYNAPSE_OAUTH_CLIENT_ID=100055
-TEAM_TO_ROLE_ARN_MAP=[{"teamId":"273957","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}]
+TEAM_TO_ROLE_ARN_MAP=[{"teamId":"3407239","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"},
```
Comment (zaro0508) [inline src/main/resources/scipooldev.properties:6]: actually, this is a java properties file so it supports comments with `#` character. https://en.wikipedia.org/wiki/.properties

let's add a few comments for this?   Can also be done in a follow on PR.

**Review by zaro0508 — APPROVED** [review]
Looks good.  A few comments would be great.

File: src/main/resources/scipooldev.properties [hunk 2]
```diff
@@ -1,5 +1,7 @@
 SYNAPSE_OAUTH_CLIENT_ID=100055
-TEAM_TO_ROLE_ARN_MAP=[{"teamId":"273957","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}]
+TEAM_TO_ROLE_ARN_MAP=[{"teamId":"3407239","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}, #scipool-admin
```
Comment (zaro0508) [inline src/main/resources/scipooldev.properties]: that's not going to work.  the comment is still inside of the json.  you need to put the comment outside, like this...

```
SYNAPSE_OAUTH_CLIENT_ID=100055
# 3407239 = scipool-admin 
# 3409012 = scipooldev-internal 
# 3409013 = scipooldev-external 
TEAM_TO_ROLE_ARN_MAP=[...]
AWS_REGION=us-east-1
```

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: src/main/resources/scipooldev.properties (hunk 2 above)
Comment (ahayden) [reply src/main/resources/scipooldev.properties]: Sorry... should have read the doc you linked.

**Pull request merged**